state/validators.md

```diff
@@ -0,0 +1,38 @@
+# Validator Selection Scheme
+
+## Abstract
+
+This document elucidates the validator selection mechanism as outlined in the [Rollkit repository](https://github.com/rollkit/rollkit). The scheme establishes an initial set of validators at the genesis of the blockchain, then leverages a priority-based system to determine the proposer of new blocks, ensuring a decentralized and fair block proposal mechanism.
```

Comment:
If https://github.com/rollkit/rollkit/issues/1272 is going to enforce a centralized sequencer for the initial `vA` version, this description change to reflect that and not mention the round-robin multi-sequencer scheme

Reply:
Updated.